build.rs

```diff
@@ -570,6 +580,7 @@ fn cc(
 
     if target.env() != "msvc" {
         let _ = c.define("_XOPEN_SOURCE", Some("700"));
+        let _ = c.flag(remap_debug_prefix());
```

Comment:
Rather than call `remap_debug_prefix()` for each compilation, let's call it once before any compilation, cache it, and pass it as an argument to `cc()` . In particular, from my experience writing makefiles it's best to avoid evaluating `env::current_dir()` more than once.

Reply:
I'm calling it once in `compile()`. But I just realized that's still called once per file. Sorry about that. I'll do another iteration on this PR.